FAQ.md

```diff
@@ -0,0 +1,57 @@
+---
+status: draft
+---
+
+# Frequently Asked Questions (FAQs)
+
+## Why create a new foundation?
+
+Though there are several foundations for open source projects to choose to join,
+each has its own focus and requirements.
```

Comment:
There are already several foundations which host open source projects, and each has its own focus and requirements.